package.json

```diff
@@ -39,32 +38,25 @@
     "babel-preset-es2015": "6.9.0",
     "babel-preset-es2016": "6.11.3",
     "babel-preset-react": "6.11.1",
+    "babel-pundle": "^1.0.0",
```

Comment:
It looks like the other requirements here are fixed versions instead of `^`, probably it's better to do the same for pundle